src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -506,9 +592,44 @@ public final class Address implements ApiMessage {
     return fieldMap;
   }
 
+  @Override
+  public String getFieldStringValue(String fieldName) throws IOException {
+    if (fieldName.equals("address")) {
+      return String.valueOf(address);
+    }
+    if (fieldName.equals("creationTimestamp")) {
+      return String.valueOf(creationTimestamp);
+    }
+    if (fieldName.equals("description")) {
+      return String.valueOf(description);
+    }
+    if (fieldName.equals("id")) {
+      return String.valueOf(id);
+    }
+    if (fieldName.equals("kind")) {
+      return String.valueOf(kind);
+    }
+    if (fieldName.equals("name")) {
+      return String.valueOf(name);
+    }
+    if (fieldName.equals("region")) {
+      return String.valueOf(region);
+    }
+    if (fieldName.equals("selfLink")) {
+      return String.valueOf(selfLink);
+    }
+    if (fieldName.equals("status")) {
+      return String.valueOf(status);
+    }
+    if (fieldName.equals("users")) {
+      return String.valueOf(users);
+    }
+    throw new IOException(String.format("Field '%s' not found in Address fields", fieldName));
```

Comment:
I think we should follow the semantics of maps in Java and return null.

Reply:
Sure thing. There just wouldn't be a way to differentiate between an actual null value for that fieldName vs the fieldName being an incorrect field name.